templates/singleton/boil_types.go.tpl

```diff
@@ -60,29 +60,81 @@ MySQL output looks like:    TableNameColNameEnumValue = "enumvalue"
 
 It only titlecases the EnumValue portion if it's snake-cased.
 */}}
+
 {{$once := onceNew}}
 {{- range $table := .Tables -}}
 	{{- range $col := $table.Columns | filterColumnsByEnum -}}
 		{{- $name := parseEnumName $col.DBType -}}
 		{{- $vals := parseEnumVals $col.DBType -}}
 		{{- $isNamed := ne (len $name) 0}}
+        {{- $enumName := "" -}}
```

Comment:
I don't really understand why this is declared here and used way further down in a completely different indentation level. Why not declare closer to where it's used?